src/scripts/helpers/backbone/views/base.coffee

```diff
@@ -117,7 +117,8 @@ define (require) ->
 
       if description isnt undefined
         $('meta[name="description"]').remove()
-        head.append("<meta name=\"description\" content=\"#{description}\">")
+        fullDescription = description + if summary isnt undefined then " " + summary else ""
```

Comment:
You may want to check for truthiness on summary rather than undefined, so that it ignores an empty string, null, and undefined.  Something like: `fullDescription = description + if summary then " #{summary}" else ''`